### PR TITLE
2.0.x chad boilerplate 2

### PR DESCRIPTION
**Pull request by crissmancd**
Additional boilerplate mixins and classes
Update to Naked theme

### CODE REVIEW

File: public/static/components/themes/naked/layouts/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<div class="rows-layout layout">
```
Comment (thescubageek) [inline public/static/components/themes/naked/layouts/index.html]: Why are we adding a custom layout to the naked theme?

**Comment by thescubageek** [comment]
@crissmancd Good refactors, just one :question: :arrow_up:

**Comment by crissmancd** [comment]
@thescubageek The naked theme is now using the first boilerplate header, which needs it's own html. The header is the only difference in the layout

**Comment by thescubageek** [comment]
@crissmancd This has been merged into 2.1.1-merge: https://github.com/g5search/g5-content-management-system/pull/769

**Pull request closed without merging**